README.md

```diff
@@ -1,2 +1,42 @@
-# stellarcli
-A command line utility for accessing the StellarStation API
+# stellar
+
+A command line utility for accessing the StellarStation API. Let's build stellar apps together!
+
+## Using the app
+
+Precompiled binaries of the app can be found on the [releases page](https://github.com/infostellarinc/stellarcli/releases).
+
+We attempt to give detailed information on using the app in the tool's help message.
+
+```bash
+$ stellar -h
+```
+
+If you see anything is unclear, feel free to file an issue or send a pull request.
+
+## Authentication
+
+The utility can be authenticated using a StellarStation API key. If you don't have one yet,
+issue a key at your account page on StellarStation Console. The key can be activated using
+
+```bash
+$ stellar auth activate-api-key path/to/stellarstation-private-key.json
```

Comment:
This might be out of scope of this PR, but how about changing command to "stellar auth activate" to be short?

Reply:
It'd be cool if we eventually have `stellar auth login` like `gcloud` :) Since it's only once, I guess it's not a huge deal to be short.